shared/uavobjectdefinition/actuatorcommand.xml

```diff
@@ -14,5 +14,8 @@
     <field defaultvalue="0" elements="1" name="MaxUpdateTime" type="uint16" units="ms">
       <description/>
     </field>
+    <field defaultvalue="0" elements="1" name="LeakyBucketContent" type="float" units="">
+      <description />
```

Comment:
??

Reply:
Yah description would be good, perhaps better name.  Good point.

Reply:
Units on both this and the other are "seconds"